app/CMakeLists.txt

```diff
@@ -351,10 +351,7 @@ add_library(firebase_app STATIC
     ${utility_HDRS}
     ${app_platform_HDRS}
     ${FIREBASE_GEN_FILE_DIR}/app/google_services_generated.h
-    memory/atomic.h
```

Comment:
Remove the old memory library's header files entirely. (It was a header-only library, with tests.)